Gopkg.toml

```diff
@@ -9,8 +9,34 @@ required = [
   "k8s.io/code-generator/cmd/informer-gen",
   "k8s.io/code-generator/cmd/openapi-gen",
   "k8s.io/gengo/args",
+  "github.com/kevinburke/go-bindata",
 ]
 
+[prune]
+  go-tests = true
+  non-go = true
+  unused-packages = true
+
+[[prune.project]]
+  name = "k8s.io/code-generator"
+  non-go = false
+  unused-packages = false
+
+[[prune.project]]
+  name = "github.com/kevinburke/go-bindata"
```

Comment:
For cluster-ingress-operator repo, we could change "github.com/kevinburke/go-bindata" dep from `[[constraint]]` to `[[prune.project]]`

Reply:
Yeah, ingress could use some cleaning up with pruning rules.